### DISCUSSION TITLE
随便拍了一张大家看看怎么样

### DISCUSSION THREAD

**Discussion by ghost** [share121/inter-knot#5359] [绝区零]
![KBGJ_1724033443425](https://github.com/user-attachments/assets/9e860316-0d56-4998-9344-400942df9230)
![KBGJ_1724033218280](https://github.com/user-attachments/assets/8145bd7b-3ae3-4b8f-9b65-a4029239aa41)

**Comment by wanbianbuliqizhong** [comment]
很不错诶，铃可爱捏
